core/src/test/scala/com/rackspace/com/papi/components/checker/handler/ApiCoverageHandlerTest.scala

```diff
@@ -0,0 +1,122 @@
+/** *
+  * Copyright 2015 Rackspace US, Inc.
+  *
+  * Licensed under the Apache License, Version 2.0 (the "License");
+  * you may not use this file except in compliance with the License.
+  * You may obtain a copy of the License at
+  *
+  * http://www.apache.org/licenses/LICENSE-2.0
+  *
+  * Unless required by applicable law or agreed to in writing, software
+  * distributed under the License is distributed on an "AS IS" BASIS,
+  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  * See the License for the specific language governing permissions and
+  * limitations under the License.
+  */
+
+package com.rackspace.com.papi.components.checker.handler
+
+import javax.servlet.FilterChain
+import javax.xml.transform.Transformer
+import javax.xml.transform.sax.SAXResult
+import javax.xml.transform.stream.StreamSource
+import com.rackspace.cloud.api.wadl.Converters._
+import com.rackspace.com.papi.components.checker.Converters._
+
+import com.rackspace.com.papi.components.checker.servlet.{CheckerServletResponse, CheckerServletRequest}
+import com.rackspace.com.papi.components.checker.step.StepHandler
+import com.rackspace.com.papi.components.checker.step.results.Result
+import com.rackspace.com.papi.components.checker.util.IdentityTransformPool
+import com.rackspace.com.papi.components.checker.{BaseValidatorSuite, TestConfig, Validator}
+import org.apache.logging.log4j.LogManager
+import org.apache.logging.log4j.core.LoggerContext
+import org.apache.logging.log4j.test.appender.ListAppender
+import org.mockito.Mockito
+import org.mockito.Mockito._
+import org.scalatest.BeforeAndAfter
+import org.scalatest.mock.MockitoSugar
+
```

Comment:
This test is not running because you're missing:

@RunWith(classOf[JUnitRunner])